railties/lib/rails/templates/rails/welcome/index.html.erb

```diff
@@ -1,9 +1,12 @@
 <!DOCTYPE html>
 <html>
 <head>
   <title>Ruby on Rails</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
+  <link rel="shortcut icon" href="<%= ruby_on_rails_logo_favicon_base64 %>" />
```

Comment:
(Oops, also applies here, sorry I didn’t include that as a suggestion as well 😅)

```suggestion
  <link rel="shortcut icon" href="<%= ruby_on_rails_logo_favicon_data_uri %>" />
```